sklearn/decomposition/pca.py

```diff
@@ -120,6 +120,10 @@ class PCA(_BasePCA):
     Notice that this class does not support sparse input. See
     :class:`TruncatedSVD` for an alternative with sparse data.
 
+    Although Bessel correction decreases the bias of the estimator, it also
```

Comment:
Sorry for being late. Could we add the link to the issue there as the discussion there makes it pretty clear? I feel we should include why we aren't including the bessel correction so as to avoid the expense of mean square error. Please let me know what you think.